### PR TITLE
[plugin.video.rocketbeans] 1.2.0

### PR DESCRIPTION
**Pull request by Razzeee**
### Description

Straight forward port to py3 mostly done with 2to3. I had to add a decode in some places manually.

I got rid of the guide function, as the website hosting the json file seems to be gone.

https://github.com/Razzeee/plugin.video.rocketbeans reflects these changes

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.rocketbeans/resources/lib/plugin.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+plugin = routing.Plugin()
+setContent(plugin.handle, 'videos')
+
+livestream_thumbnail = None
+livestream_url = None
+
+__addon__ = Addon()
+
+def getString(string_id):
+    return __addon__.getLocalizedString(string_id).encode('utf-8', 'ignore')
```
Comment (romanvm) [inline plugin.video.rocketbeans/resources/lib/plugin.py]: You don't need to do encode in Python 3 and certainly should use `ignore` error handler with great care.

File: plugin.video.rocketbeans/resources/lib/plugin.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        livestream_url, title, livestream_thumbnail = t.url, t.title, t.thumbnail
+    else:
+        livestream_url, title, livestream_thumbnail = YoutubeStream().get_live_video_info_from_channel_id(config.CHANNEL_ID)
+    return livestream_url, livestream_thumbnail, title
+
+@plugin.route('/')
+def index():
+    livestream_url, livestream_thumbnail, title = get_livestream()
+    li = createListItem(
+        ('Live | %s' % title),
```
Comment (romanvm) [inline plugin.video.rocketbeans/resources/lib/plugin.py]: This is Python 3.7, you can do f-strings :)

File: plugin.video.rocketbeans/resources/lib/plugin.py (hunk 1 above)
Comment (Razzeee) [reply plugin.video.rocketbeans/resources/lib/plugin.py]: Yeah, I don't think I needed to touch this part. Seems to still work.

File: plugin.video.rocketbeans/resources/lib/plugin.py (hunk 2 above)
Comment (Razzeee) [reply plugin.video.rocketbeans/resources/lib/plugin.py]: I just wanted to have it work. Did not really look at all the new features that are available. I'll try to keep it in mind. But I'm probably doing "low effort ports" for now.

**Comment by Razzeee** [comment]
Force pushed something in the hope of getting travis unstuck

**Pull request merged**